### DISCUSSION TITLE
视频右侧的视频推荐如何关闭

### DISCUSSION THREAD

**Discussion by Lost-Voice** [the1812/Bilibili-Evolved#4986] [提问 / Q&A]
如图，右侧的视频推荐我应该如何关闭，我记得之前应该是配置好后默认关闭的，配置如图左侧，我没动过，不知道什么时候视频推荐又出现了，我甚至不清楚是配置更新了还是bug
![image](https://github.com/user-attachments/assets/cddd43e3-c944-4a72-b1b2-85da3c574a37)

**Comment by Geeyun-JY3** [comment]
重复，请善用搜索：#4957
